src/standard.ts

```diff
@@ -1,63 +1,17 @@
-import { EngineeringNotation } from "./engineering";
-import Decimal from "break_infinity.js";
-import { toFixedEngineering } from "./utils";
+import { Notation } from "./notation";
+import type Decimal from "break_infinity.js";
+import { toFixedEngineering, abbreviate } from "./utils";
 
-const ABBREVIATIONS = [
-  "", "K", "M", "B", "T", "Qa", "Qt", "Sx", "Sp", "Oc", "No", "Dc", "UDc", "DDc",
-  "TDc", "QaDc", "QtDc", "SxDc", "SpDc", "ODc", "NDc", "Vg", "UVg", "DVg", "TVg",
-  "QaVg", "QtVg", "SxVg", "SpVg", "OVg", "NVg", "Tg", "UTg", "DTg", "TTg", "QaTg",
-  "QtTg", "SxTg", "SpTg", "OTg", "NTg", "Qd", "UQd", "DQd", "TQd", "QaQd", "QtQd",
-  "SxQd", "SpQd", "OQd", "NQd", "Qi", "UQi", "DQi", "TQi", "QaQi", "QtQi", "SxQi",
-  "SpQi", "OQi", "NQi", "Se", "USe", "DSe", "TSe", "QaSe", "QtSe", "SxSe", "SpSe",
-  "OSe", "NSe", "St", "USt", "DSt", "TSt", "QaSt", "QtSt", "SxSt", "SpSt", "OSt",
-  "NSt", "Og", "UOg", "DOg", "TOg", "QaOg", "QtOg", "SxOg", "SpOg", "OOg", "NOg",
-  "Nn", "UNn", "DNn", "TNn", "QaNn", "QtNn", "SxNn", "SpNn", "ONn", "NNn", "Ce"
-];
-
-const PREFIXES = [
-  ["", "U", "D", "T", "Qa", "Qt", "Sx", "Sp", "O", "N"],
-  ["", "Dc", "Vg", "Tg", "Qd", "Qi", "Se", "St", "Og", "Nn"],
-  ["", "Ce", "Dn", "Tc", "Qe", "Qu", "Sc", "Si", "Oe", "Ne"]
-];
-
-const PREFIXES_2 = ["", "MI-", "MC-", "NA-", "PC-", "FM-"];
-
-export class StandardNotation extends EngineeringNotation {
-  public get name(): string {
-    return "Standard";
-  }
+export class StandardNotation extends Notation {
+  public readonly name = "Standard";
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'name' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)